previews/azure-functions-service-bus.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Azure Functions with Azure Service Bus
+component: ASBFunctions
+summary: Azure Functions Preview
+related:
+ - samples/previews/azure-functions
+reviewed: 2020-06-18
+---
+
+Host NServiceBus endpoints with [Azure Functions](https://docs.microsoft.com/en-us/azure/azure-functions/).
+
+## Basic Usage
+
+### Endpoint configuration
+
+snippet: endpoint-configuration
+
+The endpoint is automatically configured with the endpoint name, the transport connection string, and the logger passed into the function using a static factory method provided by `ServiceBusTriggeredEndpointConfiguration.FromAttributes` method.
+
+Alternatively, the endpoint name can be passed in manually:
+
+snippet: alternative-endpoint-setup
+
+### Azure Function definition
+
+snippet: function-definition
+
+## Configuration
+
+## License
+
+To provide a license for the endpoints hosted as Azure Functions, the environment varialbe (Function setting) `NSERVICEBUS_LICENSE` should be used contain license as escaped text.
+For local development, `local.settings.json` can be used. In Azure, specify a Function setting using environment variable as the key.
```

Comment:
for local development, the regular licensing methods would probably work better as you don't have to escape the whole string.

Reply:
Someone who's working with Functions might find it simpler to have the environment set up in a way their other environments work.